internal/core/converter/tensorflow/option.go

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright (C) 2019 Vdaas.org Vald team ( kpango, kmrmt, rinx )
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package tensorflow provides implementation of Go API for extract data to vector
+package tensorflow
+
+import (
+	tf "github.com/tensorflow/tensorflow/tensorflow/go"
```

Comment:
unused import

Reply:
you must be next generation go compiler right?

Reply:
nooooooooo! i'm not sure about go...

Reply:
fixed